src/Components/Web/src/Forms/ModelBinding/CascadingModelBinder.cs

```diff
@@ -0,0 +1,90 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Reflection.Metadata;
+using Microsoft.AspNetCore.Components.Forms.ModelBinding;
+using Microsoft.AspNetCore.Components.Rendering;
+
+namespace Microsoft.AspNetCore.Components.Forms;
+
+/// <summary>
+/// Defines the binding context for data bound from external sources.
+/// </summary>
+public sealed class CascadingModelBinder : ICascadingValueSupplier, IComponent
```

Comment:
This only deals with forms, so we should find it a better name. We should also avoid the term "model binder". I suggest `Form(Model?)Mapper` or similar

Reply:
I agree. Was just trying to avoid making too many changes to the key parts of the developer experience without coordinating with you first.

Reply:
Updated name as per discussion.